packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js

```diff
@@ -60,4 +61,14 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
             .to.equal(JSON.stringify(fixture.state.fixtureContents, null, 2));
     });
   });
+
+  it('should set splitpane onChange to localstorage lib set', function() {
```

Comment:
How about more humane test cases? E.g. "should save splitpane position in local storage on change"